### PR TITLE
Updated video tooltips and labels.

### PR DESCRIPTION
**Pull request by dsjen**
@brianhw @jab5569 @mulby @lamagnifica

### CODE REVIEW

File: acceptance_tests/test_course_engagement.py [hunk 1]
```diff
@@ -212,7 +212,7 @@ class CourseEngagementVideoTimelineTests(CourseEngagementVideoMixin, WebAppTest)
     expected_heading = u'What were the viewing patterns for this video?'
     expected_tooltip = u'The number of students who watched each segment of the video, and the ' \
                        u'number of replays for each segment.'
-    expected_table_columns = [u'Time', u'Unique Viewers', u'Replays']
+    expected_table_columns = [u'Time', u'Segment Views', u'Segment Replays']
```
Comment (lamagnifica) [inline acceptance_tests/test_course_engagement.py]: do we still want "Unique" in here @shnayder ?

File: analytics_dashboard/static/js/engagement-video-timeline-main.js [hunk 2]
```diff
@@ -12,14 +12,14 @@ require(['vendor/domReady!', 'load/init-page'], function(doc, page) {
                 timelineSettings = [
                     {
                         key: 'num_users',
-                        title: gettext('Unique Viewers'),
+                        title: gettext('Segment Views'),
```
Comment (lamagnifica) [inline analytics_dashboard/static/js/engagement-video-timeline-main.js]: ditto on "Unique" @shnayder

**Comment by lamagnifica** [comment]
terrific, one question to Victor in line.

File: acceptance_tests/test_course_engagement.py [hunk 3]
```diff
@@ -166,10 +166,10 @@ def _test_heading_question(self):
 @skipUnless(ENABLE_COURSE_API, 'Course API must be enabled to test the video pages.')
 class CourseEngagementVideoContentTests(CourseEngagementVideoMixin, WebAppTest):
     expected_heading = u'How did students interact with course videos?'
-    expected_tooltip = u'Each bar shows the counts of complete and incomplete plays of the videos in that section. ' \
+    expected_tooltip = u'Each bar shows the counts of complete and incomplete views of the videos in that section. ' \
                        u'Click on bars with low totals or a high incomplete rate to drill down and understand ' \
                        u'why.'
-    expected_table_columns = [u'Order', u'Section Name', u'Videos', u'Started Watching', u'Finished Watching',
+    expected_table_columns = [u'Order', u'Section Name', u'Videos', u'Complete Views', u'Incomplete Views',
```
Comment (brianhw) [inline acceptance_tests/test_course_engagement.py:172]: Just wondering if the order of these matters.  I'd expect 'Started Watching' => 'Incomplete Views' and 'Finished Watching' => 'Complete Views'.  Yet this is a test, and I don't see the corresponding change in the regular code.  So I'm assuming this is just catching up with changes made in the regular code already.  Right?

**Comment by brianhw** [comment]
One clarification, otherwise LGTM. :+1:

File: acceptance_tests/test_course_engagement.py (hunk 3 above)
Comment (dsjen) [inline acceptance_tests/test_course_engagement.py:172]: I just ran the acceptance tests again, and they're indeed passing; I'm a little perplexed as to how it got out of sync.  Yes, this does get it back in order though.

File: acceptance_tests/test_course_engagement.py (hunk 1 above)
Comment (shnayder) [inline acceptance_tests/test_course_engagement.py]: Yes. Otherwise it's not clear what the difference is. I'd leave it as 'unique viewers'

Comment (dsjen) [inline acceptance_tests/test_course_engagement.py]: Ok, I will revert change.  The table columns will then be: Time, Unique Viewers, and Replays.

**Pull request merged**